man/man8/tcpconnect.8

```diff
@@ -25,6 +25,9 @@ Print usage message.
 \-t
 Include a timestamp column.
 .TP
+\-c
+Count connects per dest ip/port.
```

Comment:
Could you use "connections" here, which is more common usage, instead of "connects"? Same for below several other cases.

Reply:
"connects" may be more accurate than connections in this context? because we inspect the connect actions, not any already established connections.

Reply:
We could use "new connections" or "connects". I guess "connect" okay which is short and also captures the essence.